presto-teradata-functions/src/main/java/com/facebook/presto/teradata/functions/TeradataStringFunctions.java

```diff
@@ -38,6 +39,24 @@ public static long index(@SqlType(StandardTypes.VARCHAR) Slice string, @SqlType(
         return StringFunctions.stringPosition(string, substring);
     }
 
+    @Description("suffix starting at given index")
+    @ScalarFunction
+    @LiteralParameters("x")
+    @SqlType("varchar(x)")
+    public static Slice substring(@SqlType("varchar(x)") Slice utf8, @SqlType(StandardTypes.BIGINT) long start)
+    {
+        return StringFunctions.substr(utf8, start);
```

Comment:
Why don't you add this as an alias in StringFunctions.java instead of TeradataStringFunctions? Is there an issue with making it more generic?